src/main/java/core/basesyntax/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
```

Comment:
According to the checklist, don't keep all logic in a single package. It would be better to place the `DataConverter` interface in a dedicated package, such as `service` or `service.converter`, to maintain a clean and organized project structure.